### DISCUSSION TITLE
Ubuntu install instructions missing 'wget' call

### DISCUSSION THREAD

**Discussion by mtmail** [osm-search/Nominatim#2817] [Q&A] [answered]
The source file
https://github.com/osm-search/Nominatim/blob/master/vagrant/Install-on-Ubuntu-18.sh
contains
```
# Get the source code from Github and change into the source directory
#
if [ "x$1" == "xyes" ]; then  #DOCS:    :::sh
    cd $USERHOME
    git clone --recursive https://github.com/openstreetmap/Nominatim.git
    cd Nominatim
else                               #DOCS:
    cd $USERHOME/Nominatim         #DOCS:
fi                                 #DOCS:
```

and the final documentation page
https://nominatim.org/release-docs/latest/appendix/Install-on-Ubuntu-18/
contains
```
Get the source code from Github and change into the source directory

cd $USERHOME
wget https://nominatim.org/release/Nominatim-4.1.0.tar.bz2
tar xf Nominatim-4.1.0.tar.bz2
```

I understand the `#DOCS` gets removed with `docs/bash2md.sh` but where is the logic replacing the`git clone` command with the release URL?

The Ubuntu 18 instructions list the `wget`. For 20 and 22 that's missing.

**Comment by lonvia** [comment, marked as answer]
Release docs are adapted manually and can be found in the `docs-x.x.x` branches.

**Reply by mtmail** [reply]
Thanks a lot. I created a PR now https://github.com/osm-search/Nominatim/pull/2827
